shapleqclient/admin.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, config: QConfig, logger: logging.Logger):
         self.logger = logger
 
     def setup(self):
-        self._client.connect_to_broker(self._client.config.get_broker_address(), self._client.config.get_broker_port())
+        self._client.connect_to_broker(self._client.config.get_bootstrap_servers())
```

Comment:
We need specify keyword that one of both `bootstrap servers` and `zk quorum`

Reply:
Fixed